bash/export_recipe_env.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Unfortunately for now it made sense to do it this way, just for ease of running during builds

Reply:
One day we'll get build execution into python